assignments/arrays.js

```diff
@@ -63,32 +63,67 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+const findThirtyThree = inventory.find(item => item.id === 33);
+console.log(findThirtyThree.car_year, findThirtyThree.car_make, findThirtyThree.car_model);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+let lastCar = inventory.slice(-1)[0];
+console.log(lastCar.car_make, lastCar.car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
```

Comment:
While this works, it returns all the inventory with the models sorted. One way to sort the models is to iterate the inventory array to pluck the models, then chaining the sort() method.